server/data/mappers/inPrisonEducationMapper.ts

```diff
@@ -19,4 +19,40 @@ const toInPrisonEducation = (apiLearnerEducation: LearnerEducation): InPrisonEdu
   }
 }
 
-export default toInPrisonEducation
+/**
```

Comment:
Hopefully my doc here is sufficient and will prevent someone tripping over this in the future

Reply:
It's very good 🙌